app/javascript/src/components/Dashboard/Notes/Item.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Dropdown, Avatar, Tooltip } from "neetoui";
+
+const Item = ({ note }) => (
+  <div className="m-3 rounded-sm border border-gray-200 p-4 text-gray-500 shadow-md">
+    <div className="flex justify-between">
+      <Typography style="h3">{note.title}</Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <li>Edit</li>
+        <li>Delete</li>
+      </Dropdown>
+    </div>
+    <Typography className="mb-3" style="body1">
+      {note.description}
+    </Typography>
+    <hr />
+    <div className="mt-4 flex justify-between">
+      <div className="note-tags">
+        {note.tags.map((tag, index) => (
+          <Tag
+            className="bg-gray-100 text-gray-500"
+            key={index}
+            label={tag}
+            style="secondary"
+          />
+        ))}
+      </div>
+      <Tooltip content="Wednesday, 10:30 AM" position="bottom-start">
+        <div className="flex items-center space-x-2">
+          <Clock size="15" />
+          <Typography className="flex items-start " style="body1">
+            {note.status} 2 hours ago
```

Comment:
The time shouldn't be hardcoded. If you intend to fix it in a later PR. Please add the comment: `// TODO: <the functionality you will implement in future>`. Make sure that you remove these comments afterward.